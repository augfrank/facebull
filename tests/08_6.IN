M0   C0   C1  2798
M1   C0   C3  2729
M2   C0   C4  2997
M3   C0   C5  2797
M4   C0   C6  2550
M5   C0   C7  2562
M6   C1   C0  2858
M7   C1   C3  2709
M8   C1   C4  2026
M9   C1   C5  2738
M10   C1   C6  2186
M11   C1   C7  2113
M12   C2   C1  2590
M13   C2   C4  2851
M14   C2   C5  2172
M15   C2   C6  2102
M16   C2   C7  2691
M17   C3   C0  2750
M18   C3   C1  2082
M19   C3   C2  2902
M20   C3   C4  2492
M21   C3   C7  2250
M22   C4   C0  2929
M23   C4   C2  2968
M24   C4   C3  2986
M25   C4   C5  2218
M26   C4   C7  2721
M27   C5   C1  2754
M28   C5   C4  2736
M29   C5   C7  2334
M30   C6   C1  2015
M31   C6   C3  2785
M32   C6   C5  2355
M33   C6   C7  2939
M34   C7   C0  2501
M35   C7   C1  2096
M36   C7   C3  2363
M37   C7   C4  2247
M38   C7   C5  2476
M39   C7   C6  2094
