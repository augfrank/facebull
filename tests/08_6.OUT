18827
1 8 15 19 25 29 30 34
